### PR TITLE
Add useatom

### PR DESCRIPTION
**Pull request by pago**
#### What changed in this PR:

Introduce a `useAtom` hook. Documentation: https://melody-js.gitbook.io/docs/melody-hooks-api/useatom

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "lerna": "2.0.0-rc.4",
-  "version": "1.1.0",
+  "version": "1.2.0-0",
```
Comment (pago) [inline lerna.json]: Not sure where these changes are coming from...

**Comment by pago** [comment]
@malte-wessel Could you take a look at this?

File: packages/melody-hooks/src/hooks/useAtom.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export const useAtom = initialState => {
+    // create an state hook for storing the current value
+    const [value, setValue] = useState(initialState);
+    // useRef is a generic tool for storing mutable state that does not
+    // cause an update to happen upon change
+    const s = useRef(null);
+    // the accessor function can be memoized and will always return the
+    // the value from the ref
+    const getValue = useCallback(() => s.current);
```
Comment (malte-wessel) [inline packages/melody-hooks/src/hooks/useAtom.js]: With the changes in #80 it should be `const getValue = useCallback(() => s.current, []);`

File: packages/melody-hooks/src/util/hooks.js [hunk 3]
```diff
@@ -23,6 +23,13 @@ export const setCurrentComponent = c => {
     currentComponent = c;
 };
 
+export const getCurrentComponent = () => {
```
Comment (malte-wessel) [inline packages/melody-hooks/src/util/hooks.js]: What is the reason for adding this function? I don't see any usage of it.

**Comment by pago** [comment]
@malte-wessel Could you please take another look? Should be ready now.

**Pull request merged**